flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/plan/rules/logical/PushFilterIntoTableSourceScanRule.scala

```diff
@@ -112,8 +112,8 @@ class PushFilterIntoTableSourceScanRule extends RelOptRule(
       call.transformTo(newScan)
     } else {
       relBuilder.push(scan)
-      val converter = new RexNodeConverter(relBuilder)
```

Comment:
One question is whether `UnsolvedExpression` or `ResolvedExpression` is passed to `FilterableTableSource`.